core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -1554,7 +1556,9 @@ public Expression implementResult(AggContext info,
       SqlAggFunction aggregation = info.aggregation();
       final Method method = (aggregation == BIT_AND
           ? BuiltInMethod.BIT_AND
-          : BuiltInMethod.BIT_OR).method;
+          : (aggregation == BIT_OR
```

Comment:
It would be nice if this is simplified :)

Reply:
How about just use if else or switch.

The current nested structure becomes less readable

Reply:
Agree. Simple if else or switch should be fine.

Reply:
Good idea. Thank you, I have updated.